python/doc/theory/numerical_methods/nataf_transformation.rst

```diff
@@ -68,21 +61,20 @@ We denote by :math:`E_{\vect{\mu},\vect{\sigma},\mat{R},\psi}` the
 cumulative distribution function of the elliptical distribution
 :math:`\cE_{\vect{\mu},\vect{\sigma}, \mat{R},\psi}`.
 
-An *elliptical copula* :math:`C^E_{\mat{R},\psi}` is the copula of an
-elliptical distribution
-:math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}`.
+- An *elliptical copula* :math:`C^E_{\mat{R},\psi}` is the copula of an
+elliptical distribution :math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}`.
 
-The *generic elliptical representative* of an elliptical distribution
+- The *generic elliptical representative* of an elliptical distribution
 family :math:`\cE_{\vect{\mu},\vect{\sigma},\mat{R},\psi}` is the
 elliptical distribution whose cumulative distribution function is
 :math:`E_{\vect{0},\vect{1},\mat{R},\psi}`.
 
```

Comment:
Shall this blank line be deleted to fix the error message: "nataf_transformation.rst:65:Bullet list ends without a blank line;"?

Reply:
Thank you. I try once more...